### PR TITLE
Add a product logo next to the heading #2218

### PR DESCRIPTION
**Pull request by zahardev**
💾 [Build file](https://www.dropbox.com/scl/fi/jb5s54gkie1xzic07xzql/gravityview-2.32-b6dbb5831.zip?rlkey=hj1vdrh63b52ys2fhm6fo4znx&dl=1) (b6dbb5831).

### CODE REVIEW

File: assets/css/scss/admin-global.scss [hunk 1]
```diff
@@ -61,6 +61,43 @@
...
+		}
+		&:after {
+			content: "";
+			position: absolute;
+			display: block;
+			left: -20px;
```
Comment (zackkatz) [inline assets/css/scss/admin-global.scss:87]: @zahardev Please add a comment about why `-20px` is what is being used, just to make it clear for future people. I assume it's admin-content padding.

Comment (zahardev) [reply assets/css/scss/admin-global.scss]: Done!

**Comment by mrcasual** [comment]
@zahardev, looks good, thanks! While you're at it, could you also please dequeue notices and remove the footer, just like in GravityCalendar and GravityCharts?

**Comment by zahardev** [comment]
@mrcasual Sure, will be done. 
Did you notice the failed tests BTW? I believe they’re unrelated to the PR. Didn’t investigate it deeply, but it seems the issue might be caused by a change in the date format:

1) GVCommon_Test::test_format_date
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'4 weeks ago'
+'4 weeks from now'

**Comment by mrcasual** [comment]
> @mrcasual Sure, will be done. Did you notice the failed tests BTW? I believe they’re unrelated to the PR. Didn’t investigate it deeply, but it seems the issue might be caused by a change in the date format:
> 
> 1. GVCommon_Test::test_format_date
>    Failed asserting that two strings are equal.
>    --- Expected
>    +++ Actual
>    @@ @@
>    -'4 weeks ago'
>    +'4 weeks from now'

@zahardev, this is a flaky test that usually works but not always. It's on our to-do list (prior attempts to consistently reproduce it for a fix have failed). If you're willing to help, we'd appreciate it.

**Comment by zahardev** [comment]
> @zahardev, this is a flaky test that usually works but not always. It's on our to-do list (prior attempts to consistently reproduce it for a fix have failed). If you're willing to help, we'd appreciate it.

@mrcasual If you have failed to reproduce it, I'm not sure I can succeed at it, but I can try :) Do you want me to check this out after the CSV issue is complete, or just add it to my to-do list with low priority?

**Comment by mrcasual** [comment]
@zahardev, to be honest, we didn't spend much time on this. If I recall correctly, this test was introduced or started failing after [this commit](https://github.com/GravityKit/GravityView/commit/51757fcc4f5e5d944485a6656408f4be78841b5e). It happens once in a blue moon, but eventually, we need to resolve it. Feel free to add it to your to-do list and address it when you have a chance.

**Pull request merged**